src/irmin-pack/unix/dispatcher.ml

```diff
@@ -28,10 +28,6 @@ module Make (Fm : File_manager.S with module Io = Io.Unix) :
   module Errs = Fm.Errs
   module Control = Fm.Control
 
-  let read_suffix = ref 0
-  let read_prefix = ref 0
-  (*TODO move them in stats*)
```

Comment:
Have these moved? I don't see them elsewhere this PR.

Reply:
Do they have not, but they are currently totally unused and they will be reimplemented properly when we upgrade the replay stats